src/EditorFeatures/CSharp/BraceMatching/CSharpJsonBraceMatcher.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.Implementation.BraceMatching;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.BraceMatching
+{
+    [ExportBraceMatcher(LanguageNames.CSharp)]
+    internal class CSharpJsonBraceMatcher : IBraceMatcher
+    {
+        public Task<BraceMatchingResult?> FindBracesAsync(Document document, int position, CancellationToken cancellationToken)
+            => CommonJsonBraceMatcher.FindBracesAsync(document, position, cancellationToken);
+    }
+}
```

Comment:
In testing this, I noticed that VsVim must be implementing its own brace matching logic... 
If I have a mismatched brace inside a string, the VsVim command for jumping to the other side (`%`) jumps to the wrong location. I'll let Jared know.